### DISCUSSION TITLE
Docs - table of contents?

### DISCUSSION THREAD

**Discussion by nikfp** [VonHeikemen/lsp-zero.nvim#110] [Ideas]
After starting #108 it occurred to me that it might be helpful to have a table of contents, similar to what Mason is doing. Do you think this is a good idea? I can start working in it if so under a new PR.

**Comment by VonHeikemen** [comment]
Sounds about right. The readme is getting a little long now.

**Comment by nikfp** [comment]
Would a viable alternative to this be to break up the base Readme a bit more? For example, the API section could move to `advanced usage` OR could be it's own document. By slimming that first page down it might be easier to grok for newcomers. What do you think? (Or do both?)

**Reply by VonHeikemen** [reply]
I don't like the idea taking the api out of the main page. I feel like if I hide it behind a click might as well not even exists. There are lots of issues and discussions that could have been avoided if users would just read Advance Usage page.
